### ISSUE TITLE
Link Mobilityn rajapinta uudistuu.

### ISSUE THREAD

**Issue by johannaraisa** [KohaSuomi/Koha#1381]
### Tukipyyntö

Link Mobilityltä ilmoittivat, että vanha rajapinta tulee lopettamaan toimintansa ja näiden osalta on siirryttävä käyttämään MyLink APIa.

**Comment by johannaraisa** [comment]
Tämä on nyt testattavissa OUTIn testillä. @pirliis ja @piiaseme, voitteko tehdä testiviestejä menemään oikeisiin numeroihin? Ilmoittakaa kun on viestejä, niin ajan lähetyksen.

**Comment by pirliis** [comment]
Nyt OUTIn testillä parilla asiakkaalla tekstiviestit odottavat-tilassa:
borrowernumber=65686
borrowernumber=110280

**Comment by johannaraisa** [comment]
> Nyt OUTIn testillä parilla asiakkaalla tekstiviestit odottavat-tilassa: borrowernumber=65686 borrowernumber=110280

Lähetin viestit äsken, viestit pitäisi olla perillä.

**Comment by johannaraisa** [comment]
> > Nyt OUTIn testillä parilla asiakkaalla tekstiviestit odottavat-tilassa: borrowernumber=65686 borrowernumber=110280
> 
> Lähetin viestit äsken, viestit pitäisi olla perillä.

Tai en ole varma, rajapinta otti ne vastaan, mutta portaalissa ei näy että tänään olisi lähetetty mitään.

**Comment by pirliis** [comment]
Asiakkaan 110280 varausilmoitus on tullut tänään tekstarina klo 8.20. Sori, asiakkaalle 65686 oli jäänyt väärä matkapuhelin nro. :)

**Comment by johannaraisa** [comment]
> Asiakkaan 110280 varausilmoitus on tullut tänään tekstarina klo 8.20. Sori, asiakkaalle 65686 oli jäänyt väärä matkapuhelin nro. :)

Joo, minullekin tuli kun kokeilin. Voin laittaa sen 65686 viestin uudestaan, jos vaihdat numeron. :)

**Comment by pirliis** [comment]
> > Asiakkaan 110280 varausilmoitus on tullut tänään tekstarina klo 8.20. Sori, asiakkaalle 65686 oli jäänyt väärä matkapuhelin nro. :)
> 
> Joo, minullekin tuli kun kokeilin. Voin laittaa sen 65686 viestin uudestaan, jos vaihdat numeron. :)

Korjasin sen nyt, jos haluat testata uudestaan. :)

**Comment by pirliis** [comment]
> > > Asiakkaan 110280 varausilmoitus on tullut tänään tekstarina klo 8.20. Sori, asiakkaalle 65686 oli jäänyt väärä matkapuhelin nro. :)
> > 
> > 
> > Joo, minullekin tuli kun kokeilin. Voin laittaa sen 65686 viestin uudestaan, jos vaihdat numeron. :)
> 
> Korjasin sen nyt, jos haluat testata uudestaan. :)

Tuli tekstari-ilmoitus klo 9.03.

**Comment by johannaraisa** [comment]
Tämä siis toimii ja Oulun osalta vaihto voidaan tehdä, koska sopimus on kunnossa. Olisiko hyvä päivä vaihtoon 1.4., silloin katkaistaan vanhan laskutus?

**Comment by pirliis** [comment]
> Tämä siis toimii ja Oulun osalta vaihto voidaan tehdä, koska sopimus on kunnossa. Olisiko hyvä päivä vaihtoon 1.4., silloin katkaistaan vanhan laskutus?

Tiistaina 1.4. vaihto sopii. Varmistin asian meidän ICT-palvelupäälliköltä.

**Comment by johannaraisa** [comment]
Tämä löytyy nyt täältä, https://github.com/KohaSuomi/koha-plugin-sms-send-link-mobility-driver/tree/24.xx

**Comment by piiaseme** [comment]
Meille tuli eilen viestin LINK Mobilityltä, että varmenne vaihtuu 10.4. Välitän viestin Kohan Support-lootaan.

**Comment by johannaraisa** [comment]
Liittyy siihen vanhaan rajapintaan ja ei sekään vaadi mitään toimia.

**Comment by johannaraisa** [comment]
Nyt on OUTI:lle vaihdettu uusi ajuri, MyLink::Driver. Katselen miten lähtee sujumaan.

**Comment by piiaseme** [comment]
Minulle tuli tekstiviesti varauksesta klo 9.30 eli näyttäisi toimivan.

**Comment by johannaraisa** [comment]
Suljen tämän ja avaan Lapin käyttöönotosta uuden tiketin.
